frontend/integration-tests/tests/devconsole/git-import-flow.scenario.ts

```diff
@@ -0,0 +1,56 @@
+import { browser, $, element, by, ExpectedConditions as until } from 'protractor';
+import { appHost, checkLogs, checkErrors } from '../../protractor.conf';
+import {
+  navigateImportFromGit,
+  setBuilderImage,
+  enterGitRepoUrl,
+  addApplication,
+  applicationName,
+  appName,
+  builderImage,
+  buildImageVersion,
+} from '../../views/devconsole-view/git-imort-flow';
+import { newApplicationName, newAppName } from '../../views/devconsole-view/new-app-name.view';
+import { switchPerspective, Perspective } from '../../views/devconsole-view/dev-perspective.view';
+import { sideHeader } from '../../tests/devconsole/dev-perspective.scenario';
+
+export const version = element(by.id('8-RHOAR-link'));
+
+describe('git import flow', () => {
+  let newApplication;
+  let newApp;
+  const importFromGitHeader = element(by.id('resource-title'));
+  const createButton = $('.pf-c-button.pf-m-primary');
+
+  beforeAll(async () => {
+    await browser.get(`${appHost}/k8s/cluster/projects`);
+    newApplication = newApplicationName();
+    newApp = newAppName();
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('public git normal flow', async () => {
+    await switchPerspective(Perspective.Developer);
+    expect(sideHeader.getText()).toContain('Developer');
+    await navigateImportFromGit();
+    await browser.wait(until.textToBePresentInElement(importFromGitHeader, 'Import from git'));
+    expect(importFromGitHeader.getText()).toContain('Import from git');
+    await enterGitRepoUrl('https://github.com/sclorg/nodejs-ex.git');
+    expect(importFromGitHeader.getText()).toContain('Import from git');
+    await addApplication(newApplication, newApp);
+    expect(applicationName.getAttribute('value')).toContain(newApplication);
+    expect(appName.getAttribute('value')).toContain(newApp);
```

Comment:
Can we also check if the name field id auto-filled by repository name before entering the new name?

Reply:
But is it required as sometimes the auto-filled name does not follow the naming convention so it is better to repopulate it

Reply:
I suggesting this because auto-filled is a feature and it can be tested by this test. Or write another test to check if auto-filled is works fine or not.

>  sometimes the auto-filled name does not follow the naming convention

And it was a bug and it has been fixed.

Reply:
Can you tell me what convention it follows for auto-filling it

Reply:
name should be start with letters and it should not contain special character or dot.